lighty-modules/lighty-gnmi/lighty-gnmi-sb/README.md

```diff
@@ -55,3 +65,164 @@ This encoding enforcement does not apply to SetRequest/GetResponse targeting sca
+        final DOMDataBroker domDataBroker = service.get();
+```
+
+5. Get openconfig interfaces data from gNMI device.
+```java
+        final YangInstanceIdentifier interfacesYIID = YangInstanceIdentifier.builder().node(INTERFACES_QNAME).build();
+        final Optional<NormalizedNode<?, ?>> normalizedNode;
+        try (DOMDataTreeReadTransaction domDataTreeReadTransaction = domDataBroker.newReadOnlyTransaction()) {
+            normalizedNode = domDataTreeReadTransaction.read(LogicalDatastoreType.CONFIGURATION, interfacesYIID).get();
+        }
+        assertTrue(normalizedNode.isPresent());
+        assertEquals(INTERFACES_QNAME, normalizedNode.get().getIdentifier().getNodeType());
+```
+
+6. Set data modeled by [gnmi-test-model](../lighty-gnmi-test/src/test/resources/models/plugin_models/gnmi-test-model.yang) to gNMI device
+```java
+        final YangInstanceIdentifier testLeafListYIID = YangInstanceIdentifier.builder().node(TEST_DATA_CONTAINER_QN).build();
+        final ContainerNode testDataContainerNode = getTestDataContainerNode();
+        final DOMDataTreeWriteTransaction writeTransaction = domDataBroker.newWriteOnlyTransaction();
+        writeTransaction.put(LogicalDatastoreType.CONFIGURATION, testLeafListYIID, testDataContainerNode);
+        writeTransaction.commit().get();
+```
+
+7. Update data in gNMI device
+```java
+        final YangInstanceIdentifier testLeafListYIID = YangInstanceIdentifier.builder()
+                .node(TEST_DATA_CONTAINER_QN).build();
+        final ContainerNode updateTestDataContainerNode = getUpdateTestDataContainerNode();
+        final DOMDataTreeWriteTransaction writeTransaction = domDataBroker.newWriteOnlyTransaction();
+        writeTransaction.merge(LogicalDatastoreType.CONFIGURATION, testLeafListYIID, updateTestDataContainerNode);
+        writeTransaction.commit().get();
+```
+
+8. Delete data from gNMI device
+```java
+        final YangInstanceIdentifier testLeafListYIID = YangInstanceIdentifier.builder().node(TEST_DATA_CONTAINER_QN).build();
+        final DOMDataTreeWriteTransaction writeTransaction = domDataBroker.newWriteOnlyTransaction();
+        writeTransaction.delete(LogicalDatastoreType.CONFIGURATION, testLeafListYIID);
+        writeTransaction.commit().get();
+```
+
+## lighty.io gNMI south-bound with lighty.io Controller register certificates
+This example will show how to programmatically add certificates for lighty gNMI. Full example can be found inside test
+[GnmiWithoutRestconfTest](../lighty-gnmi-test/src/test/java/io/lighty/modules/gnmi/test/gnmi/GnmiWithoutRestconfTest.java).
+
+1. Adding certificates to keystore doesn't require device. So this example use only lighty Controller and lighty gNMI modules.
+    Certificates could be assigned to device, with [keystore-id](https://github.com/PANTHEONtech/lighty/blob/master/lighty-models/lighty-gnmi-models/lighty-gnmi-topology-model/src/main/yang/gnmi-topology.yang#L45)
+    parameter when is creating device mountpoint.
```

Comment:
`is creating device mountpoint` -> `the mountpoint is being created`

Reply:
done